app/src/main/java/kernbeisser/Windows/PermissionManagement/PermissionController.java

```diff
@@ -114,7 +114,7 @@ void loadSolutions() {
             .collect(Collectors.toCollection(ArrayList::new)));
     List<PermissionKey> values =
         Arrays.stream(selected.getKeys())
-            .filter(key -> selected == PermissionKeyGroups.ACTIONS || key.name().endsWith("_WRITE"))
+            .filter(key -> selected == PermissionKeyGroups.ACTIONS || !key.name().endsWith("_WRITE"))
```

Comment:
Das war früher so:
```
        Arrays.stream(
                PermissionKey.with(
                    e ->
                        e.getClazz() != null
                            && (e.getClazz().equals(getView().getCategory())
                                & !e.name().endsWith("_WRITE"))))
```
Frage: Ist es evtl. besser, konkret `key.name().endsWith("_READ")` zu filtern?

Reply:
Auf jeden Fall gäbe es side effects, weil names.endsWith("_READ") nicht,
[ADD_*
REMOVE_*
EDIT_*](https://github.com/heintzl/Kernbeisser-Gradle/blob/38c290213a27c135b3ab307b29462e673989398f/app/src/main/java/kernbeisser/Security/PermissionKeyGroups.java#L54-L57) 
mit inbegreifen würde.
Wahrscheinlich müssten das eigentlich Aktions sein

Reply:
Ich habe nachgeschaut, dann werden ADD, EDIT, und REMOVE nicht mehr angezeigt und können nicht mehr bearbeitet werden.

Reply:
Dann bleibt's so